modules/110-istio/images/operator-v1x21x6/werf.inc.yaml

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
Make the same edits as in images/operator-v1x19x7/werf.inc.yaml

Reply:
Done